### ISSUE TITLE
Create Endpoint for Revalidation

### ISSUE THREAD

**Issue by ogrady** [FlussuferOrga/ts-gw2-verifyBot#24]
Have a POST endpoint that triggers revalidation in two modes:

- `lazy`, to only revalidate users whose audit due date has passed (default behaviour as it is now).
- `force`, which revalidates _all_ users.

**Comment by Xyaren** [comment]
Maybe allow validating users by accountname an/or a pattern based on the accountname.

**Comment by Xyaren** [comment]
Would this be additional to the existing scheduled validation ?

**Comment by ogrady** [comment]
Yes, I think treating the TS-Bot as an independent component that can revalidate automatically on its own without the need for an external trigger is a good idea.
